src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -217,18 +218,20 @@ public Rotation2d getFieldRelativeHeading() {
   }
 
   public Rotation2d getPitch() {
-    return Rotation2d.fromDegrees(m_pigeon.getPitch());
+    // THE PIGEON'S ROLL IS THE ROBOT'S PITCH! THIS SHOULD BE getRoll()!
+    return Rotation2d.fromDegrees(m_pigeon.getRoll());
   }
   
   public Rotation2d getRoll() {
-    return Rotation2d.fromDegrees(m_pigeon.getRoll());
+    // THE PIGEON'S PITCH IS THE ROBOT'S ROLL! THIS SHOULD BE getPitch()!
```

Comment:
Should we use m_pigeon.configMountPose(PositiveX /*forward*/, PositiveZ /*up*/) during initialization to correct for it?

Reply:
I think y is positive, looking at the robot.